run.py

```
@@ -93,9 +101,30 @@ def get_tests_for_suite(suite, config):
         if f.is_file() and f.suffix == config["suffix"]
     )
 
-def run_benchmark(executable, executable_arguments, test_file, score_metric, iterations, index, total, suppress_output=False):
+def supports_wasm_benchmark_timings(executable):
+    result = subprocess.run([executable, "--help"], stdout=subprocess.PIPE, stderr=subprocess.STDOUT, text=True)
```

Ruff (0.16.0)

[error] `subprocess` call: check for execution of untrusted input

(S603)